src/iso19111/c_api.cpp

```diff
@@ -1169,15 +1201,14 @@ PJ_OBJ_LIST *proj_get_non_deprecated(PJ_CONTEXT *ctx, const PJ *obj) {
 static int proj_is_equivalent_to_internal(PJ_CONTEXT *ctx, const PJ *obj,
                                           const PJ *other,
                                           PJ_COMPARISON_CRITERION criterion) {
-    assert(obj);
-    assert(other);
-    if (!obj->iso_obj) {
+
+    if (!obj || !other){
+        proj_log_error(ctx, __FUNCTION__, "missing required input");
```

Comment:
As raised by https://travis-ci.com/github/OSGeo/PROJ/jobs/352928854, ctx is nullptr when called from proj_is_equivalent_to()

Reply:
so proj_log_error() should only be called if (ctx )